packages/cli/src/workflows/workflow.service.ee.ts

```diff
@@ -19,8 +19,8 @@ import { SharedWorkflowRepository } from '@db/repositories/sharedWorkflow.reposi
 import { BadRequestError } from '@/errors/response-errors/bad-request.error';
 import { NotFoundError } from '@/errors/response-errors/not-found.error';
 
-export class EEWorkflowsService extends WorkflowsService {
-	static async isOwned(
+export class EnterpriseWorkflowService extends WorkflowService {
```

Comment:
This needs a `@Service()` for `Container.get(EnterpriseWorkflowService)` to work